deps/npm/node_modules/request/node_modules/qs/Readme.md

```diff
@@ -0,0 +1,233 @@
+# qs
```

Comment:
I had some weird problems with this exact file recently trying to change my main working branch from `v1.x` to `master` and it wasn't to do with this patch, I didn't end up working it out but had to do some hackery to make git happy with the switch. Odd.